test/odbc/psqlodbc/test_pg_datatype_float.cpp

```diff
@@ -0,0 +1,955 @@
+#include <gtest/gtest.h>
+#include <sqlext.h>
+#include "../src/drivers.h"
+#include "../src/odbc_handler.h"
+#include "../src/query_generator.h"
+#include <string>
+#include <math.h>
+
+using std::pair;
+
+const string TABLE_NAME = "master_dbo.float_table_odbc_test";
+const string VIEW_NAME = "master_dbo.float_view_odbc_test";
+const string DATATYPE = "sys.float";
+const int NUM_COLS = 2;
+const string COL_NAMES[NUM_COLS] = {"pk", "float"};
+const int COL_LENGTH[NUM_COLS]={10,308};
+const int COL_SCALE[NUM_COLS] = {0, 308};
+
+// unsure of how these values are calculated, but they match with postgres' (non-sys) decimal type
+// Postgres documentation just says they're bytes size are variable
```

Comment:
Sorry, which values are you talking about?

Reply:
this should refer to function Colattributes, for 8 and 17 in this case for float